From: Peter Krempa <pkrempa@redhat.com>
Date: Tue, 26 Aug 2025 13:57:42 +0200
Subject: daemon: Drop log level of VIR_ERR_NO_SUPPORT to debug

The error code signals that the API the user called is not supported by
the driver. This can happen with some hypervisor drivers which don't
have everything implemented yet. There's no point in spamming the log
with it.

Closes: https://gitlab.com/libvirt/libvirt/-/issues/805
Signed-off-by: Peter Krempa <pkrempa@redhat.com>
Reviewed-by: Martin Kletzander <mkletzan@redhat.com>
(cherry picked from commit 37a1bd945899308d1c071bb885e5d1d9529d6b85)

Bug-Debian: https://bugs.debian.org/1110963

Forwarded: not-needed
Origin: https://gitlab.com/libvirt/libvirt/-/commits/37a1bd945899308d1c071bb885e5d1d9529d6b85
---
 src/remote/remote_daemon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/remote/remote_daemon.c b/src/remote/remote_daemon.c
index 1424d4c..2973813 100644
--- a/src/remote/remote_daemon.c
+++ b/src/remote/remote_daemon.c
@@ -108,6 +108,7 @@ static int daemonErrorLogFilter(virErrorPtr err, int priority)
     case VIR_ERR_NO_CLIENT:
     case VIR_ERR_NO_HOSTNAME:
     case VIR_ERR_NO_NETWORK_METADATA:
+    case VIR_ERR_NO_SUPPORT:
         return VIR_LOG_DEBUG;
     }
 
